pithos/plugins/notification_icon.py

```diff
@@ -201,6 +206,13 @@ def context_menu(self, widget, button, time, data=None):
                 data.show_all()
                 data.popup(None, None, None, None, 3, time)
 
+    def _on_icon_changed(self, prefs_dialog, icon):
+        self.settings['data'] = icon
+        if indicator_capable:
+            self.ind.set_icon_full(icon, '')
```

Comment:
The 2nd arg is the "icon description" whatever the hell that's suppose to mean? "Well it's sort of a vase looking pot with a music note on top of it."...